state_machine/src/scripts/interactive_folding.py

```diff
@@ -0,0 +1,371 @@
+#!/usr/bin/env python
```

Comment:
what is the difference with state_machine/scr/interactive_folding.py?

Reply:
I yeeted that folder, it was just a zombie